Sources/Testing/Running/Configuration.TestFilter.swift

```diff
@@ -28,42 +28,52 @@ extension Configuration {
       case excluding
     }
 
-    /// An enumeration describing the various kinds of test filter.
-    fileprivate enum Kind: Sendable {
+    /// An enumeration describing a predicate that a test filter may specify.
+    fileprivate enum Predicate: Sendable, Codable {
       /// The test filter has no effect.
       ///
-      /// All tests are allowed when passed to a test filter with this kind.
-      case unfiltered
+      /// All tests are allowed when passed to a test filter with this
+      /// predicate.
+      case all
```

Comment:
Is it necessary to rename this case? (It's not a problem _per se_ but it makes the diff more complicated.)